geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/HScanIntegrationTest.java

```diff
@@ -64,27 +59,4 @@ public void givenCursorLessThanIntMinValue_returnsCursorError() {
     assertThatThrownBy(() -> jedis.hscan("a", tooSmallCursor.toString()))
         .hasMessageContaining(ERROR_CURSOR);
   }
-
-
-  @Test
-  public void givenCount_shouldReturnExpectedNumberOfEntries() {
-    Map<byte[], byte[]> entryMap = new HashMap<>();
-    entryMap.put("1".getBytes(), "yellow".getBytes());
-    entryMap.put("2".getBytes(), "green".getBytes());
-    entryMap.put("3".getBytes(), "orange".getBytes());
-    jedis.hmset("colors".getBytes(), entryMap);
-
-    int COUNT_PARAM = 2;
-
-    ScanParams scanParams = new ScanParams();
-    scanParams.count(COUNT_PARAM);
-    ScanResult<Map.Entry<byte[], byte[]>> result;
-
-    String cursor = "0";
-
-    result = jedis.hscan("colors".getBytes(), cursor.getBytes(), scanParams);
-
-    assertThat(result.getResult().size()).isEqualTo(COUNT_PARAM);
-  }
-
```

Comment:
hscan is allowed to return more than `count` number of entries, due to hash collisions. This test would now fail with both native redis and radish.